lib/ansible/modules/cloud/ovirt/ovirt_cluster.py

```diff
@@ -65,8 +65,9 @@
     gluster:
         description:
             - "If I(True), hosts in this cluster will be used as Gluster Storage
-               server nodes, and not for running virtual machines."
-            - "By default the cluster is created for virtual machine hosts."
+               server nodes."
+            - "By default the cluster is created for virtual machine hosts only."
+            - "Note that you cannot add oVirt Node (RHV-H) to a Gluster-enabled cluster"
```

Comment:
Sandro, is above sentence correct?

@sandrobonazzola

Reply:
Please wait on this, it looks like this sentence that I found on docs, is not true. ""Note that you cannot add oVirt Node (RHV-H) to a Gluster-enabled cluster""

I 'll get back here when I get answer.

Reply:
@KKoukiou Hi, just wondering if you got an answer to this.

Reply:
@gundalow notreally AFAIK. But I 'll close the PR since it's stale long time.